packages/legend-query/src/components/QueryBuilderExplorerPanel.tsx

```diff
@@ -802,6 +811,16 @@ export const QueryBuilderExplorerPanel = observer(
       explorerState.setHumanizePropertyName(
         !explorerState.humanizePropertyName,
       );
+    useEffect(() => {
+      flowResult(
+        queryBuilderState.querySetupState.analyzeMappingModelCoverage(),
+      ).catch(applicationStore.alertUnhandledError);
+    }, [
+      applicationStore,
```

Comment:
do we need to really watch for all the elements in this list? I felt like we don't need `explorerState`